src/utilities/colorToTailwind.tsx

```diff
@@ -0,0 +1,12 @@
+export default function colorToTailwind(color: string) {
```

Comment:
This file does not need to have the `tsx` suffix, because it does not contain suffix.

Reply:
Confirmed fixed